ci/.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: RSpec CI
+on: [pull_request, push]
+jobs:
+  test:
+    name: Ruby ${{ matrix.ruby }}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        ruby:
+          - 3.0.0-preview1
+          - 2.7
+          - 2.6
+          - 2.5
+          - 2.4
+          - 2.3
+          - 2.2
+          - 2.1.9
+          - ruby-head
+      fail-fast: false
+    continue-on-error: ${{ matrix.ruby == 'jruby-9.2.13.0' || endsWith(matrix.ruby, 'head') }}
```

Comment:
Is it still needed?

Reply:
head at least is needed post 3.0.0's release, and the jruby one is needed until its fixed